simple_slurm/scancel.py

```diff
@@ -0,0 +1,56 @@
+import subprocess
+from loguru import logger
```

Comment:
Should be declared as dependency right?

Reply:
I personaly like using loguru, but maybe it's not wanted. But I doubt this PR will be merged anyway :-S